kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Worker.java

```diff
@@ -51,14 +50,14 @@
 import org.apache.kafka.connect.sink.SinkTaskContext;
 
 public class Worker extends Channel {
```

Comment:
Just FYI, it doesn't look like you have rebased on top of latest `main`

Reply:
Fixed.